docs/data-dictionary/types/saneringskrediet_lopende_rente.yml

```diff
@@ -0,0 +1,6 @@
+type: number
```

Comment:
@Frank-SBN Is lopende rente een gegeven of een verschil tussen vorderingen en betalingen?

Reply:
Helder. Alleen relevant bij uitbetaling neem ik aan?